src/libieee1284-3-windows-header.patch

```diff
@@ -0,0 +1,91 @@
+From d8da09469573051c2fb85da1048bef5a4742cb2f Mon Sep 17 00:00:00 2001
+From: =?UTF-8?q?Daniel=20Gl=C3=B6ckner?= <daniel-gl@gmx.net>
+Date: Sun, 20 Sep 2015 19:15:15 +0200
+Subject: [PATCH] search for windows.h
+
+instead of relying on the __CYGWIN__ macro
+---
+ configure.in     | 25 ++++++++++---------------
+ src/access_lpt.c |  2 +-
+ src/detect.c     |  2 +-
+ src/ports.c      |  2 +-
+ 4 files changed, 13 insertions(+), 18 deletions(-)
+
+diff --git a/configure.in b/configure.in
+index 7cd9561..9b6b82a 100644
+--- a/configure.in
++++ b/configure.in
+@@ -37,21 +37,16 @@ case "{$host}" in
+         AC_DEFINE(HAVE_SOLARIS,1,enable solaris iop access)
+ 	solaris_io=true
+ ;;
+-*-*-cygwin*)
+-        dnl Strip version number from uname and make sure we're on NT not 9x
+-        case `uname -s | sed 's/-.*$//'` in
+-        CYGWIN_NT*)
+-                AC_CHECK_HEADER(w32api/windows.h, [
+-      	                AC_DEFINE(HAVE_CYGWIN_NT,1,enable w32api access)
+-	        ], AC_MSG_ERROR(You need the cygwin w32api files on NT)) 
+-        ;;
+-        *)
+-      	        AC_DEFINE(HAVE_CYGWIN_9X,1,enable win95 access)
+-        ;;
+-        esac
+-;;
+-*-*-mingw*)
+-dnl        AC_DEFINE(HAVE_CYGWIN_9X,1,enable win95 access)
++*-*-cygwin* | *-*-mingw*)
++	windows_h_found=false
++	AC_CHECK_HEADERS([windows.h] [w32api/windows.h], [
++		windows_h_found=true
++		break
++	])
++	AS_VAR_IF(windows_h_found, [false], [
++		AC_MSG_ERROR(windows.h not found)
++	])
++	AC_DEFINE(HAVE_CYGWIN_9X,1,enable win95 access)
```

Comment:
This case construct had inconsistent tab/space indentation to begin with. Just look at line 20 of the patch, which is also indented with a tab. The following lines for the Python support even combine tab and space for fine grained indentation to the opening brace. The lines added by my patch are consistent in that they only use tab.